calculator/src/Main.java

```diff
@@ -1,16 +1,16 @@
 // Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
 // then press Enter. You can now see whitespace characters in your code.
-public class Main extends sub {
+public class Main extends add {
     public static void main(String[] args) {
 
         calculator(10,16);
-        subtract(2,3);
+        additions(2,3);
     }
     public static void calculator(int firstNumber, int secondNumber){
 
         int Answer = firstNumber + secondNumber;
 
-        System.out.print("Answer: " +Answer);
+        System.out.println("Answer: " +Answer);
         //yes bro
```

Comment:
Could you please remove this comment yes bro.

Reply:
Okay, Thanks Amin. Thank you for spotting that. i will look into it and get it resolve

Reply:
@SalisAmin All issue has been resolved. Please have a look again and get back to me. Thanks